### ISSUE TITLE
Bug when loading PCN

### ISSUE THREAD

**Issue by royhaast** [amarquand/PCNtoolkit#90]
Hi,

During the (local) installation, I encountered an error when trying to load the PCN module in Python:

```
>>> import pcntoolkit as pcn
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ROBARTS/rhaast/PCNtoolkit/pcntoolkit/__init__.py", line 1, in <module>
    from . import trendsurf
  File "/home/ROBARTS/rhaast/PCNtoolkit/pcntoolkit/trendsurf.py", line 21, in <module>
    from pcntoolkit.model.bayesreg import BLR
  File "/home/ROBARTS/rhaast/PCNtoolkit/pcntoolkit/model/__init__.py", line 6, in <module>
    from . import hbr
  File "<fstring>", line 1
    (dist=)
```

I was able to fix this by removing `=` from the expression in the curly braces on lines https://github.com/amarquand/PCNtoolkit/blob/10a9928b586fc49fa8f9b988efb3ed8f29e5bfb4/pcntoolkit/model/hbr.py#L449
and
https://github.com/amarquand/PCNtoolkit/blob/10a9928b586fc49fa8f9b988efb3ed8f29e5bfb4/pcntoolkit/model/hbr.py#L450

After this, the PCN module loads fine.
Roy

**Comment by AuguB** [comment]
Hi,

The syntax I used there is part of the python 3.8 f-string syntax: https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging

I didn't realize this would be a problem, but it's an easy fix, as you pointed out. We will probably re-write it for backwards compatibility. Updating to python 3.8 would also solve the problem of course. 

Stijn

**Comment by AuguB** [comment]
I just resolved this issue within version 0.24. All changes are backwards compatible. Closing this.
